### PR TITLE
Node bump: latest Dubnium — 10.22.0

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Bumps node version to 10.22, which has some security patches.

https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#10.22.0

## Does this change need to be reproduced in dotcom-rendering ?

- [X] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?

Latest and greatest node LTS 10.x

### Tested

- [X] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-10.16
+v10.22.0
```
Comment (jfsoul) [inline .nvmrc:1]: I assume nvm is agnostic about whether we include `v` here or not?

Comment (mxdvl) [reply .nvmrc:1]: Yes, both nvm and nfm are agnostic. Maybe it should be fixed to the `v10.22`

I just think that by convention its a little clearer to use the **v** prefix.

Comment (mxdvl) [reply .nvmrc:1]: I can also confirm [the TeamCity build picks it up][tc].

[tc]: https://teamcity.gutools.co.uk/viewLog.html?buildId=566033&buildTypeId=dotcom_master&tab=buildLog&_focus=91&state=103%2C86#_state=103,86

**Comment by SiAdcock** [comment]
Came here to find out what Dubnium means. 

TIL it's the [codename for Node v10](https://nodejs.org/en/about/releases/) 💡

**Comment by mxdvl** [comment]
@SiAdcock what’s your take on Node v14 LTS naming? Fermium, Flerovium, Fluorine, Francium? I think _Fluorine_ would be _clean_. Assuming they keep the alphabetical elements numbering going on…

https://www.lenntech.com/periodic/name/alphabetic.htm

**Comment by SiAdcock** [comment]
Intersting! I'm not knowledgable enough to have an opinion on that 😅 

@AWare might have a take though 😄

**Comment by mxdvl** [comment]
Tested on `CODE`: everything works.

**Review by jfsoul — APPROVED** [review]

**Pull request merged**